src/AbstractEnum.php

```diff
@@ -34,9 +34,6 @@
         $enumTypes = static::getEnumTypes();
         $enums = [];
 
-        if (!is_array($enumTypes)) {
-            $enumTypes = [$enumTypes];
-        }
```

Comment:
Technically, yes.

According to the `AbstractEnum::getEnumTypes`, no because it should return an array, not a string.

After re-flexion, I think @roukmoute did this because most of the time, the enum will concern only one type.

I can do it smoother by:

* Keeping this conversion
* Update `AbstractEnum::getEnumTypes` phpdoc to allow `string|string[]`
* Add a test case to increase the coverage

Would you like a try?